activerecord/test/cases/adapters/postgresql/enum_test.rb

```diff
@@ -98,6 +98,15 @@ def test_assigning_enum_to_nil
     assert_nil model.reload.current_mood
   end
 
+  def test_enum_without_keyword_arguments
+    klass = Class.new(ActiveRecord::Base) do
+      self.table_name = "postgresql_enums"
+      enum :current_mood, [:sad, :ok, :happy]
+    end
+
+    assert_equal "happy", klass.create!(current_mood: :happy).current_mood
+  end
```

Comment:
I'm not sure I understand the purpose of this test (how is it different than the generic test added in `enum_test.rb`?)

Reply:
This uses the native `enum` type in Postgres